entries/h/hva.nl.json

```
@@ -0,0 +1,13 @@
+{
+  "Site Name": {
+    "domain": "hva.nl",
+    "tfa": [
+      "call",
+      "totp"
+    ],
+    "documentation": "https://student.hva.nl/az-lemmas/studenten/hva-breed/its-si/tweestapsverificatie/2-stapsverificatie.html",
+    "categories": [
+      "universities"
+    ]
```

Since the site is only relevant in the Netherlands, I recommend a regions array.
```suggestion
    ],
    "regions": [
      "nl"
    ]
```